test/groundwater.jl

```diff
@@ -4,14 +4,14 @@ function initial_head(x)
 end
 
 """
-    transient_aquifer_1d(x, time, conductivity, specific_yield, aquifer_length, Β)
+    transient_aquifer_1d(x, time, conductivity, specific_yield, aquifer_length, beta)
```

Comment:
Suggest to change `beta` to a more descriptive variable name (as the other variable names in the function).

Reply:
What does `beta` stand for in this function. Is it a slope or something else?

Reply:
Good question, I don't think it is a slope. @Huite: can you explain what `Β` does stand for?

Reply:
I agree that `beta` needs a more descriptive name.
This change is, however, out of scope for this PR.
We can tackle this in another issue / PR